scripts/golangci.yml.in

```diff
@@ -169,7 +169,12 @@ linters:
     - funlen
     - testpackage
     - exhaustivestruct
-    # TODO: enable goerr113, see: https://github.com/ceph/ceph-csi/issues/1227
+    # TODO: enable goerr113
+    # This requires extra addition of unnecessary code. Hence, we
+    # pefer to disable this linter. But, it can be enabled if we
```

Comment:
typo "pefer".

You can drop the "TODO: " line completely, the description is sufficient IMHO